Tests/RenderingTests/Layout Containers/Forms/FormTests.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  FormTests.swift
+//  
+//
+//  Created by Shadowfacts on 1/18/23.
+//
+
+import XCTest
+import SwiftUI
+
+@MainActor
+final class FormTests: XCTestCase {
+
+    func testForm() throws {
+        try assertMatch("""
```

Comment:
TextField cannot be rendered by SwiftUI's `ImageRenderer`, so this test will produce images like the following:

![FormTests swift-15-testForm()_view](https://user-images.githubusercontent.com/13581484/213259991-5fa36b25-9391-4160-bc3b-2635df591d10.png)

I can look into using a UIKit-based snapshot technique that can capture UIKit-wrapped Views in a separate PR.

Reply:
Ah, that's too bad. In that case, I'll remove this test for now until we revisit it.